Cargo.toml

```diff
@@ -36,4 +36,4 @@ cty = "0.2.1"
 env_logger = "0.7.1"
 ctrlc = "3.1.3"
 clap = "2.33.0"
-colored = "1.9.3"
+colored = "1.9.3"
```

Comment:
You don't need to change anything here but I suggest you install the editorconfig plugin. I have added an editorconfig file to the master branch which will generally make your coding experience a little bit more consistent.